backend/data_validation/tests/test_data_val_2022.py

```diff
@@ -25,10 +25,10 @@ def example_scouting_data(
     attempted_traversal: int = 1,
     climb_time: float = 15.0,
     final_climb_type: str = "Traversal",
-    defense_pct: float = 1.0,
-    defense_rating: int = 5,
-    counter_defense_pct: int = 0,
-    counter_defense_rating: int = 0,
+    defense_pct: float = 0.5,
```

Comment:
can we not change default values in `example_scouting_data()` and instead just change them in specific unit tests? it's more annoying but this is bad practice